modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -239,6 +239,43 @@ The Zipkin receiver ingests data in the Zipkin v1 and v2 formats.
 [id="processors_{context}"]
 === Processors
 
+
+[id="batch-processor_{context}"]
+==== Batch processor
+
+The Resource Detection processor is designed to identify host resource details in alignment with OpenTelemetry's resource semantic standards. Using this detected information, it can add or replace the resource values in telemetry data.
```

Comment:
Looks like a copy&paste error?

Reply:
Thank you for spotting this, Andreas.

Reply:
Sorry. It seems when I moved the PR from one branch to another I messed up